client/build.rs

```diff
@@ -0,0 +1,49 @@
+//! Build script which builds smartcontract for test
```

Comment:
Are not there any other appropriate path than `client/` ?

Reply:
Well, I can't see any alternative. Smartcontract is used in client integration tests. Moving it in some another crate (for example new `smartcontracts` crate) will make it hard to get generated wasm from client tests, cause `build.rs` output should be only in its specific crate directory.

Reply:
How about `client/tests/wasm/` ?

Reply:
I can't cause it's a module, not a crate. From this [reference](https://doc.rust-lang.org/cargo/reference/build-scripts.html):

> Placing a file named build.rs in the root of a package will cause Cargo to compile that script and execute it just before building the package.

What I can do is to create a new crate and move test into it. But should I?

Reply:
I see, being in the crate root is a requirement.
[Renaming](https://doc.rust-lang.org/cargo/reference/manifest.html#the-build-field
) is still available but I don't think it makes sense.
Let's leave it as it is